ambry-frontend/src/test/java/com/github/ambry/frontend/NamedBlobPutHandlerTest.java

```diff
@@ -670,4 +691,10 @@ private RestRequest getRestRequest(JSONObject headers, String path, byte[] reque
         RequestPath.parse(request, frontendConfig.pathPrefixesToRemove, CLUSTER_NAME));
     return request;
   }
+
+  public String stripPrefixAndExtension(String blobId) throws RestServiceException {
+    return RestUtils.stripSlashAndExtensionFromId(
+        RequestPath.parse(blobId, Collections.emptyMap(), frontendConfig.pathPrefixesToRemove, CLUSTER_NAME)
+            .getOperationOrBlobId(false));
+  }
```

Comment:
you don't need this function, this is the same in the NamedBlobPutHandler.

Reply:
Sure, previously the method is in the callbackchain only, I moved to namedBlobPutHandler so it can be re-used.